### PR TITLE
fixing code smells

### PR DESCRIPTION
**Pull request by notwillcasey**
- fixing code smells to reduce tech debt
- working to figure out the security issue with multer

### CODE REVIEW

File: api/controllers.js [hunk 1]
```diff
@@ -49,7 +49,21 @@ module.exports = {
...
     }
-    db.createNewConversation(req.body)
+    console.log(req.body);
+    return helpers.storePhoto(req)
+      .then((photoData) => {
+        let conversation = {};
+        if (photoData != null) {
+          conversation.senderId = req.body.senderId;
+          conversation.userId2 = req.body.userId2;
+          conversation.photo = photoData.Location;
```
Comment (aleksandar-cakic) [inline api/controllers.js:59]: is there a reason why Location is capitalized?

File: api/routes.js [hunk 2]
```diff
@@ -3,13 +3,15 @@ const controllers = require('./controllers');
 const auth = require('./middleware').auth;
 const helpers = require('./helpers');
 const multer = require('multer');
+const memStorage = multer.memoryStorage();
 
 const upload = multer({
-  storage: multer.memoryStorage(),
-  fileFilter: helpers.filefilter,
+  storage: memStorage,
```
Comment (aleksandar-cakic) [inline api/routes.js:9]: Reviewed and market security warning as safe

**Review by aleksandar-cakic — APPROVED** [review]

File: api/controllers.js (hunk 1 above)
Comment (notwillcasey) [reply api/controllers.js:59]: That is how s3 sends the data back. All of the keys are capitalized.

File: api/routes.js (hunk 2 above)
Comment (notwillcasey) [reply api/routes.js:9]: Thanks!

**Pull request merged**